### PR TITLE
apploader - Adds button for screenshots

### PR DESCRIPTION
**Pull request by halemmerich**
This adds the UI for https://github.com/espruino/EspruinoAppLoaderCore/pull/50

image

### CODE REVIEW

File: android.html [hunk 1]
```diff
@@ -135,16 +135,17 @@ <h2>App Loader</h2>
...
            <button class="btn tooltip" id="installdefault" data-tooltip="Delete everything, install default apps">Install default apps</button>
            <button class="btn tooltip" id="installfavourite" data-tooltip="Delete everything, install your favourites">Install favourite apps</button>
+           <button class="btn tooltip" id="defaultbanglesettings" data-tooltip="Reset your Bangle's settings to the defaults">Reset Settings</button>
         </p><p>
            <button class="btn tooltip" id="newGithubIssue" data-tooltip="Create a new issue on GitHub">New issue on GitHub</button>
-           <button class="btn tooltip" id="downloadallapps" data-tooltip="Download all Bangle.js files to a ZIP file">Backup</button>
-           <button class="btn tooltip" id="uploadallapps" data-tooltip="Restore Bangle.js from a ZIP file">Restore</button>
-           <button class="btn tooltip" id="defaultbanglesettings" data-tooltip="Reset your Bangle's settings to the defaults">Reset Settings</button>
-           <button class="btn tooltip" id="webideremote" data-tooltip="Enable the Web IDE remote server">Web IDE Remote</button>
```
[the remark this answers is not in the document]
Comment (halemmerich) [reply android.html:147]: Yes, that happened accidentally while reordering the buttons. Is that reordering itself ok for you or should I restore the original order while putting webideremote back?

[the remark this answers is not in the document]
Comment (gfwilliams) [reply android.html:147]: The re-ordering is fine - thanks!

[the remark this answers is not in the document]
Comment (halemmerich) [reply android.html:147]: Does the remote IDE need the GadgetBridge-specific JS or why is the button not available in the normal index.html?

**Comment by gfwilliams** [comment]
Thanks!

**Pull request merged**